apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -343,12 +343,70 @@ fn options_route() -> Result<Response<Full<Bytes>>, hyper::http::Error> {
     Ok(response)
 }
 
-async fn health_route() -> Result<Response<Full<Bytes>>, hyper::http::Error> {
-    let response = Response::builder()
-        .status(StatusCode::OK)
-        .body(Full::new(Bytes::from("Success")))
-        .unwrap();
-    Ok(response)
+async fn health_route(
+    project: &Project,
+    redis_client: &Arc<RedisClient>,
+) -> Result<Response<Full<Bytes>>, hyper::http::Error> {
+    // Check Redis connectivity
+    let redis_healthy = redis_client.is_connected();
+
+    // Check Redpanda/Kafka connectivity
+    let redpanda_config = project.redpanda_config.clone();
+    let kafka_healthy = match kafka::client::health_check(&redpanda_config).await {
```

Comment:
no need to clone, just do `kafka::client::health_check(&project.redpanda_config)`